docs/api.md

```diff
@@ -1621,9 +1621,20 @@ Returns an `Villager` instance which represents the trading window you are openi
 
 Uses the open `villagerInstance` to trade.
 
-#### bot.setCommandBlock(pos, command, track_output, [mode])
+#### bot.setCommandBlock(pos, command, [options])
```

Comment:
Changing the API results with a breaking change. You can leave the previous arguments as another docs entry. And mark is as deprecated. Then just check if track_output is a boolean or an object and handle it correctly.

Reply:
Though I'd just add and export the flags I described in some comments below.
And let users do:
```js
const { TRACK_OUTPUT, ALWAYS_ACTIVE } = bot.commandBlockFlags
const { MODE_REDSTONE } = bot.commandBlockModes
bot.setCommandBlock(pos, command, TRACK_OUTPUT | ALWAYS_ACTIVE, MODE_REDSTONE)
```

Reply:
I said while merging the previous pr that this mode parameter is basically not part of the API and it will get changed, so it's ok to "break" it here

Reply:
I don't understand that the mode parameter is not part of the API, I don't get it.

Another reason that supports the breaking changes is that bot.setCommandBlock() has been only working in 1.8 since its implementation untill a week ago or so, so certainly not many (if any) people are using it.

Reply:
Not sure what you mean. I said in the previous PR that it was not part of the API so it's not, that's all. That's what makes it ok to remove/change it now.